aquadoggo_cli/src/schemas.rs

```diff
@@ -15,28 +16,3 @@ pub fn read_schema_ids_from_file(file: &mut File) -> Result<Vec<SchemaId>> {
     ))?;
     Ok(value.clone().try_into::<Vec<SchemaId>>()?)
 }
-
-#[cfg(test)]
-mod tests {
-    use std::fs::{create_dir_all, File};
-    use std::io::Write;
-
-    use tempfile::TempDir;
-
-    use super::read_schema_ids_from_file;
-
-    #[test]
-    fn reads_schema_ids_from_file() {
-        let tmp_dir = TempDir::new().unwrap();
-        let mut tmp_path = tmp_dir.path().to_owned();
-        tmp_path.push("schema.toml");
-
-        create_dir_all(tmp_path.parent().unwrap()).unwrap();
-        let mut file = File::create(&tmp_path).unwrap();
-        file.write_all("schemas = [\"plant_0020c11ab63099193a8c8516cc00f88bfc8cdd657b94a99fef2fce86aaaede84f87d\"]".as_bytes()).unwrap();
-
-        let result = read_schema_ids_from_file(&mut file);
-        println!("{result:?}");
-        assert!(result.is_ok());
-    }
-}
```

Comment:
That test somehow didn't create a temporary file on my local machine (also I think the method always returns an empty array as the key is wrong in the written test data). I didn't dive deeper into troubleshooting it as this will change soon with the new configuration stack (where we probably will use serde to deserialize the `config.toml` file).